api/swagger-spec/oapi-v1.json

```diff
@@ -17703,7 +17703,7 @@
      "completionDeadlineSeconds": {
       "type": "integer",
       "format": "int64",
-      "description": "completionDeadlineSeconds is an optional duration in seconds, counted from the time when a build pod gets scheduled in the system, that the build may be active on a node before the system actively tries to terminate the build; value must be positive integer"
+      "description": "CompletionDeadlineSeconds is an optional duration in seconds, counted from the time when a build pod gets scheduled in the system, that the build may be active on a node before the system actively tries to terminate the build; value must be positive integer"
```

Comment:
all these items that got turned into uppercase need to be set back to initial lowercase (it's a new change that was introduced in types.go while your commonspec refactor was happening).